images/openstack/rdo-stein.repo

```diff
@@ -0,0 +1,6 @@
+[openstack-stein]
+name=OpenStack Stein Repository
+baseurl=http://mirror.centos.org/centos/7/cloud/$basearch/openstack-stein/
+#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=cloud-openstack-stein
+gpgcheck=0
```

Comment:
Wouldn't it be better to keep gpgcheck enabled? The https://www.rdoproject.org/repos/rdo-release.rpm rpm provides the public key for CentOS-SIG-Cloud, we could inject it into the image the same way you do for the repo file.

Reply:
sure, I was scrambling to get a patch up quickly yesterday. I didnt think of putting the CentOS-SIG-Cloud public key in a static file like the .repo file.